java-core/google-cloud-core/clirr-ignored-differences.xml

```diff
@@ -12,4 +12,12 @@
         <className>com/google/cloud/ReadChannel</className>
         <method>long limit()</method>
     </difference>
+    <!-- These methods were switched to java.time and are internally used
+    in testing -->
+    <difference>
+        <differenceType>7005</differenceType>
+        <className>com/google/cloud/testing/*</className>
+        <method>* *(org.threeten.bp.Duration)</method>
+        <to>*</to>
+    </difference>
```

Comment:
Do you know which methods were flagging this 7005 error code? This is method argument changed and I don't think that technically any method argument should have changed. Might be false positive and just want to double check.

Reply:
It is an accurate flag. This exemption is for the `com/google/cloud/testing/*` classes, although the only one that applies is `BaseEmulatorHelper`.

My approach was to simply change the signature of these methods to `java.time` because they are only meant to be used in unit testing.

Maybe we still want to use an overload approach (`${method}Duration(java.time.Duration)`) so the CLIRR check flags "new methods" instead?

Reply:
I see, I misread this. I didn't see that it was in the testing module and thought it was public methods used for testing.

Reply:
Hmm, actually on some more thought I'm a bit on the fence about this. It is indeed in a testing package and marked with InternalApi, but I believe it is also packaged publicly in a JAR that anyone case use. I doubt anyone is using this anything other than testing purposes (i.e. BaseEmulatorHelperTest). Let's see if we can get some guidance/ another opinion.

Reply:
from @blakeli0 and @lqiu96, let's treat this as any other public surface: we only _add_ overloaded methods.